Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+library 'jenkins-ptcs-library@0.6.1'
+
+def project = 'ipa-log-app'
+
+pipeline {
+    agent any
+ 
+    stage('Checkout') {
+	        checkout scm
+      }
+
+          
+    stage('Package') {
+        def published = publishContainerToGcr(project);
+
+        if(env.BRANCH_NAME == 'master') {
+            updateImageToK8sTestEnv(published);
+        }
+      }
+    }
+}
```

Comment:
missing new line on end of the file.

Reply:
okay I will fix it with the next version of the Jenkinsfile.